test/e2e/framework/test_context.go

```diff
@@ -162,7 +162,7 @@ func RegisterCommonFlags() {
 	flag.BoolVar(&TestContext.DeleteNamespace, "delete-namespace", true, "If true tests will delete namespace after completion. It is only designed to make debugging easier, DO NOT turn it off by default.")
 	flag.BoolVar(&TestContext.DeleteNamespaceOnFailure, "delete-namespace-on-failure", true, "If true, framework will delete test namespace on failure. Used only during test debugging.")
 	flag.IntVar(&TestContext.AllowedNotReadyNodes, "allowed-not-ready-nodes", 0, "If non-zero, framework will allow for that many non-ready nodes when checking for all ready nodes.")
-	flag.StringVar(&TestContext.Host, "host", "http://127.0.0.1:8080", "The host, or apiserver, to connect to")
+	flag.StringVar(&TestContext.Host, "host", "", "The host, or apiserver, to connect to")
```

Comment:
....Can you insert a  comment or just updated the flag description why the default must be empty? otherwise I'm sure it'll get accidentally added back in, as it looks like an omission